<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToDoScreen</class>
 <widget class="QWidget" name="ToDoScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ToDoScreen</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>To-Dos</string>
     </property>
     <property name="textFormat">
      <enum>Qt::MarkdownText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0">
    <widget class="ScrollWidget" name="activeTodos">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="todoInput"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="1" rowspan="2">
    <widget class="ScrollWidget" name="completedTodos">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrollWidget</class>
   <extends>QFrame</extends>
   <header location="global">widgets/scrollwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>ToDoScreen</receiver>
   <slot>createTodo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>302</x>
     <y>563</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>592</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createTodo()</slot>
 </slots>
</ui>
